### PR TITLE
Update dazzle.sh

### PR DESCRIPTION
**Pull request by michaeleino**
added argument -u and --user to enable external user definition

### CODE REVIEW

File: dazzle.sh [hunk 1]
```diff
@@ -6,6 +6,13 @@
 # To Public License, Version 2, as published by Sam Hocevar. See
 # http://sam.zoy.org/wtfpl/COPYING for more details.
 
+#added to enable external user definition
+case $2 in
+        u|--user)
```
Comment (hbons) [inline dazzle.sh]: shouldn't it be -u here?

Comment (michaeleino) [reply dazzle.sh]: yes you are right...

**Comment by michaeleino** [comment]
Actually I think it is better to use getopts, and place it in the start of the scripts

**Comment by hbons** [comment]
It's fine. It's a simple script. Thanks!

**Pull request merged**